server/api/api.go

```diff
@@ -460,6 +461,82 @@ func (a *API) handlePostBlocks(w http.ResponseWriter, r *http.Request) {
 	auditRec.Success()
 }
 
+func (a *API) handleUpdateUserConfig(w http.ResponseWriter, r *http.Request) {
+	// swagger:operation PATCH /api/v1/users/{userID}/config updateUserConfig
+	//
+	// Updates user config
+	//
+	// ---
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: userID
+	//   in: path
+	//   description: User ID
+	//   required: true
+	//   type: string
+	// - name: Body
+	//   in: body
+	//   description: User config patch to apply
+	//   required: true
+	//   schema:
+	//     "$ref": "#/definitions/UserPropPatch"
+	// security:
+	// - BearerAuth: []
+	// responses:
+	//   '200':
+	//     description: success
+	//   default:
+	//     description: internal error
+	//     schema:
+	//       "$ref": "#/definitions/ErrorResponse
+
+	requestBody, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		a.errorResponse(w, r.URL.Path, http.StatusInternalServerError, "", err)
+		return
+	}
+
+	var patch *model.UserPropPatch
+	err = json.Unmarshal(requestBody, &patch)
+	if err != nil {
+		a.errorResponse(w, r.URL.Path, http.StatusInternalServerError, "", err)
+		return
+	}
+
+	vars := mux.Vars(r)
+	userID := vars["userID"]
+
+	ctx := r.Context()
+	session := ctx.Value(sessionContextKey).(*model.Session)
+
+	auditRec := a.makeAuditRecord(r, "updateUserConfig", audit.Fail)
+	defer a.audit.LogRecord(audit.LevelRead, auditRec)
+	auditRec.AddMeta("performedBy", userID)
```

Comment:
We need to keep the fields names as consistent as possible since customers query by them.  These would be `user_id` and `session_id`.

Reply:
`user_id` and `session_id` are already being included by `LogRecord()` method as general audit metadata. The user ID and session user ID are logged here in the context of this specific operation. I know user ID is already included by `LogRecord` but I still wanna keep it here for context.

Would renaming here as `performed_by` and `performed_on` resolve your concern?

Reply:
When customers query these audit calls they know `user_id` and `session_id` answer the "who did this".  Adding duplicate fields with the same info doesn't break anything, but potentially confuses the customer.  They wonder, "can `performed_by` ever to be different than `user_id`?  Do I need to special case this API when querying?"

Reply:
Okay, that makes sense. I've removed these meta fields from here now as they are already being included in `LogRecord`.